aw_client/client.py

```diff
@@ -36,8 +36,10 @@ def __init__(self, client_name: str, testing=False) -> None:
         client_config = load_config()
         configsection = "server" if not testing else "server-testing"
 
-        self.server_hostname = client_config[configsection]["hostname"]
-        self.server_port = client_config[configsection]["port"]
+        self.server_host = server_host if server_host != None else client_config[configsection]["hostname"]
+        self.server_port = server_port if server_port != None else client_config[configsection]["port"]
+
+        self.instance = SingleInstance("{}@{}:{}".format(self.client_name, self.server_host, self.server_port))
```

Comment:
I don't think that will be a valid in a Windows file. Just use dashes in place of `@` and `:`.

Reply:
This needs to be fixed.